lightning/src/ln/channelmonitor.rs

```diff
@@ -1284,8 +1284,8 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 
 	/// Allows this monitor to scan only for transactions which are applicable. Note that this is
 	/// optional, without it this monitor cannot be used in an SPV client, but you may wish to
-	/// avoid this (or call unset_funding_info) on a monitor you wish to send to a watchtower as it
-	/// provides slightly better privacy.
+	/// avoid this on a monitor you wish to send to a watchtower as it provides slightly better
+	/// privacy.
```

Comment:
Ehh, hopefully it doesn't cause more CPU just more blocks to fetch. Note that this is an internal function and, after this gets merged, this and a bunch of the Option<>-setting set_* fns in ChannelMonitor should just be removed wholesale (this included).

Reply:
Hmmm I think it's more CPU because you have to try to decrypt every blob_punishment_txn instead of just looking for a spend of the funding_txo. No need to add comment like you said it's internal function. Watchtowers trade-off should be documented on a higher-level.